<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCreateClass</class>
 <widget class="QMainWindow" name="QtCreateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1373</width>
    <height>1029</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtCreate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/QtCreate/Logo.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="ScheduleGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>181</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>调度策略</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="EDFpushButton">
        <property name="text">
         <string>相对短截止时间优先调度
(EDF)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RMSpushButton">
        <property name="text">
         <string>单调速率调度
(RMS)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DMSpushButton">
        <property name="text">
         <string>短截止时间调度
(DMS)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LSTpushButton">
        <property name="text">
         <string>最小松弛时间调度
(LST)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MFPSpushButton">
        <property name="text">
         <string>最小空闲时间共享分区调度
(MFPS)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ConsoleGroupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>630</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>实时调度信息</string>
    </property>
    <widget class="QTextEdit" name="ConsoleTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GantGroupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>1101</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>运行状态图</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Timeslice</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1071</width>
       <height>291</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="GantScrollContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>3500</width>
        <height>272</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>3500</width>
        <height>271</height>
       </size>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>0</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>SimSun-ExtB</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Partition</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>0</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>SimSun-ExtB</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Task</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DataGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>调度数据</string>
    </property>
    <widget class="QTextEdit" name="DataTextEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ResultGroupBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>630</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>运行结果</string>
    </property>
    <widget class="QTextEdit" name="ResultTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>241</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="FileOpenButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>910</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>FileOpen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FileSaveButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>910</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>FileSave</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>650</y>
      <width>121</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/QtCreate/logo8.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="FlightgroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>飞行控制</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>起飞前检查
(Preflight Check)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>起飞
(Departure)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>巡航
(Cruise)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>降落
(Arrival)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PanelgroupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>361</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>仪表盘</string>
    </property>
    <widget class="QVideoWidget" name="PanelWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="View1groupBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>360</width>
      <height>270</height>
     </rect>
    </property>
    <property name="title">
     <string>侧方视角</string>
    </property>
    <widget class="QVideoWidget" name="View1Widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="View2groupBox">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>360</width>
      <height>270</height>
     </rect>
    </property>
    <property name="title">
     <string>尾翼视角</string>
    </property>
    <widget class="QVideoWidget" name="View2Widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1373</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qtcreate.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>FileOpenButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>LoadAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1011</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FileSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>SaveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1007</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LSTpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>LSTBtn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>705</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RMSpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>RMSBtn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MFPSpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>MFPSBtn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>751</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EDFpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>EDFBtn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DMSpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>DMSBtn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>685</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>Departure_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>644</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>735</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>Preflight_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>742</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>Cruise_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>686</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>744</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>Arrival_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>734</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>769</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>EDFBtn_Click()</slot>
  <slot>RMSBtn_Click()</slot>
  <slot>DMSBtn_Click()</slot>
  <slot>LoadAction()</slot>
  <slot>SaveAction()</slot>
  <slot>MFPSBtn_Click()</slot>
  <slot>LSTBtn_Click()</slot>
  <slot>Departure_Click()</slot>
  <slot>Preflight_Click()</slot>
  <slot>Cruise_Click()</slot>
  <slot>Arrival_Click()</slot>
 </slots>
</ui>
